tmt/steps/provision/connect.py

```diff
@@ -2,6 +2,8 @@
 
 import tmt
 
+CONN_TIMEOUT = 60 * 4
```

Comment:
This constant is repeated several times. What about moving it to `tmt/provision/__init__.py` instead? Let's use `CONNECTION_TIMEOUT` as the name.

Reply:
Ok. Solved.